packages/cli/src/tools/config/readConfigFromDisk.js

```diff
@@ -82,7 +98,9 @@ export function readLegacyDependencyConfigFromDisk(
       hooks: config.commands,
       params: config.params,
     },
-    commands: [].concat(config.plugin || []),
+    commands: []
+      .concat(config.plugin || [])
```

Comment:
There's no need for the `[].concat`, can you clean it up while at it?

Reply:
Why not? `config.plugin` can be either an array or a string, want to cast it to array

Reply:
Wrote that before seeing comment on this, feel free to ignore